### ISSUE TITLE
EmissionUtils methods are package-private

### ISSUE THREAD

**Issue by JWJoubert** [matsim-org/matsim-libs#1255]
Is there a specific reason why the `getHbefaRoadType(...)` and `setHbefaRoadType(...)` methods in the [`EmissionUtils`](https://github.com/matsim-org/matsim-libs/blob/5a4228d2e7ad4d68ee780cf89f7223074db46f56/contribs/emissions/src/main/java/org/matsim/contrib/emissions/EmissionUtils.java#L63) class are `package-private`? 

I appreciate that one would like to reduce the public footprint, but these two methods (I believe) are important so that one need not hard-code the attribute name in one's own code. 

```
private static final String HBEFA_ROAD_TYPE = "hbefa_road_type";
```

Is it fine, @kainagel, @kt86 and @Janekdererste, if I make it public?

**Comment by kainagel** [comment]
Does not seem to make sense, I agree.  @kt86 , could you please have a look?

**Comment by kt86** [comment]
Dear Johan,
yes you are right. This can (should) be public.  So please go ahead :)
Best, Kai

**Comment by kt86** [comment]
This was solved in #1260  by @JWJoubert. --> closed
